mesmerize_napari/core.py

```diff
@@ -387,10 +387,20 @@ def get_correlation_image(self) -> np.ndarray:
         """
         path = get_full_data_path(self._series['outputs']['corr-img-path'])
         return np.load(str(path))
+    def get_pnr_image(self) -> np.ndarray:
```

Comment:
put this under `CNMFExtensions` with a `@validate('cnmf')

actually we will need to think about how to validate `cnmfe` with the same `cnmf` validator

Reply:
Just use the same validate